plugins/processor/grok/processor_grok_test.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2022 iLogtail Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package grok
+
+import (
+	"testing"
+
+	"github.com/alibaba/ilogtail/pkg/logger"
+	"github.com/alibaba/ilogtail/pkg/protocol"
+	"github.com/alibaba/ilogtail/plugins/test/mock"
+	"github.com/stretchr/testify/require"
+	"gotest.tools/assert"
+)
+
+func init() {
```

Comment:
add benchmark test to processor_grok and processor_reg， and compare their performance.

Reply:
done